lib/danger_plugin.rb

```diff
@@ -86,6 +88,29 @@ def output_report(report)
       if !threshold.nil? && (report.coverage * 100) < threshold
         fail("Code coverage under minimum of #{threshold}%")
       end
+
+      # Notify failure if minimum coverage hasn't been reached for modified/added files
+      file_threshold = args.first[:minimum_coverage_percentage_for_changed_files].to_i
+      ignore_list = args.first[:filename_prefix_ignore_list]
+      if !file_threshold.nil?
+        report.targets.each do |target|
+          target.files.each do |file|
+            if ((file.coverage * 100) < file_threshold) #check if file coverage is less than that defined
+              if !ignore_list.nil? #check is ignore list is provided or not
+                ignore_list.each do |ignore|
+                  if (!file.name.include? ignore)
```

Comment:
is this an "includes" or a "prefix match"? 
if it's a prefix match (as the parameter name `filename_prefix_ignore_list` suggests) we should check with `starts_with?` instead of `includes?`

Reply:
This is an includes/contains match, because ideally we want to be able to skip files like `SomeViewController` if the list includes `View`. Removed _prefix_ from the parameter name to avoid this confusion.